Pod/Classes/PhxPush.h

```diff
@@ -9,17 +9,21 @@
 #import <Foundation/Foundation.h>
 #import "PhxTypes.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @class PhxChannel;
 
 @interface PhxPush : NSObject
 
-- (id)initWithChannel:(PhxChannel*)channel
-                event:(NSString*)event
-              payload:(NSDictionary*)payload;
+- (instancetype)initWithChannel:(PhxChannel*)channel
+                          event:(NSString*)event
+                        payload:(NSDictionary*)payload;
 
 - (void)send;
 
-- (PhxPush*)onReceive:(NSString*)status callback:(OnMessage)callback;
-- (PhxPush*)after:(int)ms callback:(After)callback;
+- (PhxPush *)onReceive:(NSString *)status callback:(OnMessage)callback;
+- (PhxPush *)after:(NSTimeInterval)ms callback:(After)callback;
```

Comment:
no, if you review the usage, it's passed to `NSTimer` as-is, so what I actually should do is rename the parameter variable